benchmark/src/util.ts

```diff
@@ -0,0 +1,85 @@
+import { BenchmarkFunction } from "./benchmark_types";
+
+export function round(num: number, decimalPlaces: number = 0) {
+    return tonumber(string.format(`%.${decimalPlaces}f`, num));
+}
+
+export const json: {
+    decode: (this: void, str: string) => {};
+    encode: (this: void, val: any) => string;
+} = require("json");
+
+export function readFile(path: string): string | undefined {
+    const fileOpenArray = io.open(path, "rb");
+
+    if (fileOpenArray && fileOpenArray[0]) {
+        const fileHandle = fileOpenArray[0];
+        let fileContent = readAll(fileHandle);
+        fileHandle.close();
+
+        if (fileContent) {
+            return fileContent;
+        }
+    }
+}
+
+export function readAll(file: LuaFile): string | undefined {
+    let content: (string | undefined)[];
+    if (_VERSION == "Lua 5.3") {
+        // @ts-ignore
+        content = file.read("a");
+    } else {
+        // JIT
+        // @ts-ignore
+        content = file.read("*a");
+    }
+
+    if (content && content[0]) {
+        return content[0];
+    }
```

Comment:
Isn't content going to be always defined, since it's a tuple?
```suggestion
    return content[0];
```